### PR TITLE
Kotlin plugin example added and README updated for Kotlin.

### PR DESCRIPTION
**Pull request by anidotnet**

### CODE REVIEW

File: demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package ro.fortsoft.pf4j.demo.kotlin
```
Comment (decebals) [inline demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt]: Please add Copyright header. Take it from an existent java file.

Comment (anidotnet) [inline demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt]: Copyright notice added

Comment (anidotnet) [inline demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt]: If there is anything pending from my end please let me know.

File: demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Anindya Chatterjee
```
Comment (decebals) [inline demo_gradle/plugins/plugin3/src/main/kotlin/ro/fortsoft/pf4j/demo/kotlin/KotlinPlugin.kt]: `Copyright (C) 2017 the original author or authors.` ?

You are the author of this class so I don't see a problem with my propose. A copy/paste from an existed .java file is fine. Do you see a problem?

**Comment by decebals** [comment]
Thanks!

**Pull request merged**